grails-app/controllers/com/mini/asaas/user/UserController.groovy

```diff
@@ -3,14 +3,17 @@ package com.mini.asaas.user
 import grails.validation.ValidationException
 
 import grails.compiler.GrailsCompileStatic
+import grails.plugin.springsecurity.annotation.Secured
 
 @GrailsCompileStatic
+@Secured("isAuthenticated()")
 class UserController {
 
     UserService userService
 
     def index() {}
 
+    @Secured("isFullyAuthenticated()")
```

Comment:
Não funcionou dessa maneira?
```suggestion
    @Secured(['IS_AUTHENTICATED_FULLY'])
```

Reply:
Funcionou sim. Tinha visto que funciona das duas formas e preferi essa, mas posso mudar

Reply:
Tem alguma diferença real entre as duas?

Reply:
Não, são equivalentes. Vi [aqui](https://grails.github.io/grails-spring-security-core/5.3.x/index.html#urls-and-authorities) essa forma e resolvi usar

Reply:
tranquilo, pode manter dessa forma então